games-board/megamek/megamek-0.34.10-r2.ebuild

```diff
@@ -24,15 +25,14 @@ pkg_setup() {
 }
 
 src_prepare() {
-	default
-
```

Comment:
if i get it right then this one need the patched eclass so the commit message should be changed.